test/cucumber/steps/ParkingSpaceReminderSteps.groovy

```diff
@@ -0,0 +1,31 @@
+import static cucumber.api.groovy.EN.*
+
+// Cenario 1	
+Given(~'user "([^"]*)" parked on spot "([^"]*)" using the system$'){ String user, int numSpot ->

```

Comment:
Falta colocar ^, exemplo: Given(~'^user "([^"]*)" parked on spot "([^"]*)" using the system$'), ocorreu a mesma coisa no when e then